en/lessons/basics/mix.md

```diff
@@ -136,17 +136,17 @@ This is analogous to `bundle install`:
 $ mix deps.get
 ```
 
-That's it!  We've defined and fetched our project dependencies.
+That's it! We've defined and fetched our project dependencies.
 Now we're prepared to add dependencies when the time comes.
 
 ## Environments
 
 Mix, much like Bundler, supports differing environments.
 Out of the box mix works with three environments:
 
-+ `:dev` — The default environment.
-+ `:test` — Used by `mix test`.  Covered further in our next lesson.
-+ `:prod` — Used when we ship our application to production.
+- `:dev` — The default environment.
```

Comment:
Such changes in Markdown probably need to be changed in **all** the translated files?

Reply:
@lex111 I don't think it's necessary as this is just something my IDE formatter did automatically.
The result is the same :)